src/client/components/app/Transactions/Labs/LabDepositTx.tsx

```diff
@@ -48,6 +49,9 @@ export const LabDepositTx: FC<LabDepositTxProps> = ({ onClose }) => {
   const [debouncedAmount, isDebouncePending] = useDebounce(amount, 500);
   const [txCompleted, setTxCompleted] = useState(false);
   const [isFetchingAllowance, setIsFetchingAllowance] = useState(false);
+  const servicesEnabled = useAppSelector(AppSelectors.selectServicesEnabled);
+  const simulationsEnabled = servicesEnabled.tenderly;
+  const zapperEnabled = servicesEnabled.zapper;
```

Comment:
```suggestion
  const { tenderly: simulationsEnabled, zapper: zapperEnabled } = servicesEnabled;
```

Reply:
placed them in different line just because of what mentioned above about
```
const simulationsEnabled = servicesEnabled.tenderly || servicesEnabled.alternateService;
```

Reply:
I've left my opinion in the other conversation, but if we go with that for simulations, then shouldn't we do the same for zaps? As in, have a `const zapsEnabled = servicesEnabled.zapper || servicesEnabled.alternativeZap`

I still think this should live in a centralized place and not scattered around, `LabDepositTx` knows too much about what is an enabled simulation etc.

Reply:
yeah, expanded on your comment above